cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,33 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const user = {
+    username: 'Flood',
+    password: 'qwert!Q12345'
+  };
 
+  it('should provide an ability to login', () => {
+    cy.visit('/login');
+    cy.get('#userName')
+      .type(user.username);
+    cy.get('#password')
+      .type(user.password);
+    cy.get('#login').click();
+    cy.get('#userName-value')
+      .should('contain', user.username);
+    cy.url().should('include', '/profile');
   });
 
-  it('', () => {
-
+  it('should allow to search for the book', () => {
+    cy.login();
```

Comment:
You could also add user object field's to request body. And then input user object as a parameter when you call this command to fully use user object) Well done anyway

Reply:
agree